common/app/model/meta.scala

```diff
@@ -238,7 +238,8 @@ final case class MetaData (
   customSignPosting: Option[NavItem] = None,
   javascriptConfigOverrides: Map[String, JsValue] = Map(),
   opengraphPropertiesOverrides: Map[String, String] = Map(),
-  twitterPropertiesOverrides: Map[String, String] = Map()
+  twitterPropertiesOverrides: Map[String, String] = Map(),
+  isMinute: Boolean = false
```

Comment:
Again, I don't think `isMinute` is worth adding as metadata to the top-level of content taxonomy